pkg/frontend/mysql_cmd_executor.go

```diff
@@ -2493,6 +2493,8 @@ func executeStmtWithWorkspace(ses FeSession,
 	case *tree.RollbackTransaction:
 		execCtx.txnOpt.byRollback = true
 		return nil
+	case *tree.SavePoint, *tree.ReleaseSavePoint, *tree.RollbackToSavePoint:
+		return nil
```

Comment:
We should return an error for RollbackToSavePoint.   Otherwise it will cause a user app data corruption.

Just fail to rollback, force app to abort whole transaction for now.

Reply:
rollback to savepoint 改为报错。